File: c/PROBLEMS/1080.c
#include <stdio.h>
int main(){
    long long int i, e, max, pos;
    scanf("%lld", e);
    max = e;
    pos = 0;
    for(i=1;i<100;i++){
        scanf("%lld", &e);
        if(e>max){
            max = e;
            pos = i;
        }
    }
    printf("%lld\n%lld\n", max, pos+1);
    
}

File: js/PROBLEMS/1080.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')
e = parseInt(lines[0])
max = e
pos = 0

for (i=1;i<=99;i++){
    if (parseInt(lines[i]) > max){
        max = parseInt(lines[i])
        pos = i;
    }
}
console.log(max)
console.log(pos+1 + '')